mti_nma/steps/avgshape/avgshape.py

```diff
@@ -4,125 +4,85 @@
 import logging
 from pathlib import Path
 from typing import Dict, List, Optional, Union
+import pandas as pd
+from aicsshparam import aicsshtools
 
 from datastep import Step, log_run_params
+from datastep.file_utils import manifest_filepaths_rel2abs
```

Comment:
I think we don't need this for now bc it was not working properly when the file path columns is not called filepath. I will check the new version of `dataset` to see whether this was fixed.

Reply:
Yeah it is fixed - I reworked all of the filepath referencing use the manifest to read filepaths in